docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md

```diff
@@ -255,6 +255,79 @@ def sql_table(
+1. **Apply Incremental Configuration**: Enable incremental loading with your configuration's `incremental` argument.
+1. **Deduplication**: When using incremental loading, the system automatically handles the deduplication of rows based on the primary key (if available) or row hash for tables without a primary key.
+
+#### Incremental Loading Example
+1. Consider a table with a `last_modified` timestamp column. By setting this column as your cursor and specifying an 
+   initial value, the loader generates a SQL query filtering rows with `last_modified` values greater than the specified initial value.
+
+   ```python
+   from sql_database import sql_table
+   from datetime import datetime
+
+   # Example: Incrementally loading a table based on a timestamp column
+   table = sql_table(
+       table='your_table_name',
+       incremental=dlt.sources.incremental(
+           'last_modified',  # Cursor column name
+           initial_value=datetime(2024, 1, 1)  # Initial cursor value
+       )
+   )
+
+   info = pipeline.extract(table, write_disposition="merge")  
+   print(info)  
+   ```
+
+1. To incrementally load the "family" table using the sql_database source method:
+
+   ```python
+   source = sql_database().with_resources("family")
+   #using the "updated" field as an incremental field using initial value of January 1, 2022, at midnight
+   source.family.apply_hints(incremental=dlt.sources.incremental("updated"),initial_value=pendulum.DateTime(2022, 1, 1, 0, 0, 0))
+   #running the pipeline
+   info = pipeline.run(source, write_disposition="merge")
+   print(info)
+   ```
+   In this example, we load data from the `family` table, using the `updated` column for incremental loading. In the first run, the process loads all data starting from midnight (00:00:00) on January 1, 2022. Subsequent runs perform incremental loading, guided by the values in the `updated` field.
+
+1. To incrementally load the "family" table using the 'sql_table' resource.
+
+   ```python
+   family = sql_table(
+       table="family",
+       incremental=dlt.sources.incremental(
+           "updated", initial_value=pendulum.datetime(2022, 1, 1, 0, 0, 0)
+       ),
+   )
+   # Running the pipeline
+   info = pipeline.extract(family, write_disposition="merge")
+   print(info)
+   ```
+
+   This process initially loads all data from the `family` table starting at midnight on January 1, 2022. For later runs, it uses the `updated` field for incremental loading as well.
+
+   :::info
+   * For merge write disposition, the source table needs a primary key, which `dlt` automatically sets up.
+   * `apply_hints` is a powerful method that enables schema modifications after resource creation, like adjusting write disposition and primary keys. You can choose from various tables and use `apply_hints` multiple times to create pipelines with merged, appendend, or replaced resources.
+   :::
+
+:::note
```

Comment:
Let's move this note up, for example before `### Configuring Incremental Loading` section. Otherwise, it looks a bit strange now because `note` is right below the `info`
<img width="867" alt="image" src="https://github.com/dlt-hub/dlt/assets/9139429/9074cdf1-a0da-4612-9f7d-1c5183d3bc9f">

Reply:
okay